R/available.R

```diff
@@ -65,7 +65,11 @@ create <- function(name, ...) {
 #' @export
 suggest <- function(path = ".", title = NULL) {
   if (is.null(title)) {
-    title <- unname(desc::desc(path)$get("Title"))
```

Comment:
Could you suggest a less ambiguous error message? I do not find the current one ambiguous...